mlrun/track/base_tracker.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import importlib
+from abc import abstractmethod
+
+import mlrun
+
+from ..utils import logger
+from .tracker import Tracker
+
+
+class BaseTracker(Tracker):
+    TRACKED_MODULE_NAME = ...
```

Comment:
For better documentation, please add the following (sphinx will parse it nicely later on)
```suggestion
    #: The module name this tracker will look for.
    TRACKED_MODULE_NAME: str = ...
```